### PR TITLE
Added Faction as a Perk Type and fixed a bug.

### PR DESCRIPTION
**Pull request by JFranklin3000**
There was a bug on the item-perk-sheet.hbs that was creating a second value for tracking Perk type. So I fixed that and add faction as a type. I also fixed all of the Origin Perks in the Compendium.

### CODE REVIEW

**Review by phildominguez — APPROVED** [review]

**Review by phildominguez — CHANGES_REQUESTED** [review]
Migration, as discussed

File: module/essence20.mjs [hunk 1]
```diff
@@ -30,7 +30,7 @@ function runMigrations() {
     return;
   }
 
-  const NEEDS_MIGRATION_VERSION = game.system.flags.needsMigrationVersion
+  const NEEDS_MIGRATION_VERSION = 4
```
Comment (phildominguez) [inline module/essence20.mjs:33]:
```suggestion
  const NEEDS_MIGRATION_VERSION = game.system.flags.needsMigrationVersion;
```

File: module/migration.mjs [hunk 2]
```diff
@@ -22,6 +22,7 @@ export const migrateWorld = async function() {
         await actor.update({"system.skills.-=social": null});
         await actor.update({"system.skills.-=speed": null});
         await actor.update({"system.skills.-=any": null});
+
```
Comment (phildominguez) [inline module/migration.mjs:25]:
```suggestion
```

File: module/migration.mjs [hunk 3]
```diff
@@ -248,6 +255,7 @@ export const migrateCompendium = async function(pack) {
           break;
         case "Item":
           updateData = migrateItemData(doc.toObject());
+
```
Comment (phildominguez) [inline module/migration.mjs:258]:
```suggestion
```
How are you adding these random newlines lol

File: module/migration.mjs [hunk 4]
```diff
@@ -90,7 +92,7 @@ export const migrateWorld = async function() {
   }
 
   // Set the migration as complete
-  game.settings.set("essence20", "systemMigrationVersion", game.system.version);
+  // game.settings.set("essence20", "systemMigrationVersion", game.system.version);
```
Comment (phildominguez) [inline module/migration.mjs:95]:
```suggestion
  game.settings.set("essence20", "systemMigrationVersion", game.system.version);
```

File: module/migration.mjs [hunk 5]
```diff
@@ -40,6 +41,7 @@ export const migrateWorld = async function() {
         console.log(`Migrating Item document ${item.name}`);
         console.log(updateData);
         await item.update(updateData, {enforceTypes: false, diff: valid});
+        await item.update({"system.-=perkType": null});
```
Comment (phildominguez) [inline module/migration.mjs:44]: I'd put this with the other perk logic below

Comment (JFranklin3000) [reply module/migration.mjs:43]: I have it there so we don't delete it until the item is updated in case there are any issues with the migration.

**Review by guswadsworth12 — APPROVED** [review]
Works good and looks good.

**Review by phildominguez — APPROVED** [review]

**Pull request merged**